package.json

```diff
@@ -110,7 +113,7 @@
     "karma-notify-reporter": "^1.3.0",
     "karma-parallel": "^0.3.1",
     "karma-typescript": "^5.5.1",
-    "netlify-lambda": "^2.0.6",
+    "netlify-lambda": "1.4.5",
```

Comment:
Why was this downgraded to the older version?

Reply:
It's not a downgrade it the same version we have on the `next` branch. I think this package was just updated by mistake